minard/shifter_information.py

```diff
@@ -35,11 +35,12 @@ def get_experts():
     Returns a list of the names of all on-call experts.
     """
     conn = engine.connect()
-    result = conn.execute("SELECT firstname FROM experts")
+    result = conn.execute("SELECT firstname, lastname FROM experts")
     row = result.fetchall()
     names = []
-    for name in row:
-        names.append((name[0], name[0].capitalize()))
+    for first, last in row:
+        name = first.capitalize() + " " + last.capitalize()
```

Comment:
Can you just store the name capitalized in the database? This won't handle "LaTorre" for example.